libcni/api_test.go

```diff
@@ -1513,5 +1584,82 @@ var _ = Describe("Invoking plugins", func() {
 			Expect(foundFirst).To(BeTrue())
 			Expect(foundSecond).To(BeTrue())
 		})
+
+		It("creates separate config cache files for multiple attachments to the same network", func() {
```

Comment:
Shouldn't there be just one test now for both cached config and result? Rather than two here.